src/main.rs

```diff
@@ -16,7 +16,7 @@ use std::hash::Hasher;
 use std::io::{self, Read, Write};
 use std::path::{PathBuf, Path};
 
-use cargo::core::{Workspace, GitReference, SourceId};
+use cargo::core::{Workspace, GitReference, SourceId, maybe_allow_nightly_features};
```

Comment:
I wonder if this should perhaps use https://docs.rs/cargo/0.31.0/cargo/core/fn.enable_nightly_features.html instead? I'm fine leaving nightly checks to Cargo and `cargo vendor` should basically strive to be as usable as possible in as many situations as possible. Do you think that's ok?

Reply:
Sure, that could work too.